datadog/resource_datadog_monitor.go

```diff
@@ -331,7 +332,15 @@ func resourceDatadogMonitorRead(d *schema.ResourceData, meta interface{}) error
 	d.Set("timeout_h", m.Options.GetTimeoutH())
 	d.Set("escalation_message", m.Options.GetEscalationMessage())
 	d.Set("silenced", m.Options.Silenced)
-	d.Set("include_tags", m.Options.GetIncludeTags())
+	if includeTags, ok := m.Options.GetIncludeTagsOk(); ok {
```

Comment:
I believe the real issue is here: https://github.com/terraform-providers/terraform-provider-datadog/blob/74b24b9c212b3691e7fa055d37aea96cdd17edb5/vendor/github.com/zorkian/go-datadog-api/datadog-accessors.go#L10560

Reply:
Using `GetIncludeTagsOk()` seems the right thing to do right now.